tests/unit/modules/test_gpg.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import os
+import shutil
+
+# Posix only
+try:
+    import gpg
+except ImportError:
+    pass
+
+
+# Import Salt Testing libs
+from tests.support.helpers import destructiveTest
+from tests.support.unit import TestCase, skipIf
+from tests.support.paths import TMP
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import (
+    NO_MOCK,
+    NO_MOCK_REASON,
+    MagicMock,
+    patch
+)
+
+# Import Salt libs
+import salt.utils.platform
+import salt.utils.files
+import salt.modules.gpg as gpg
```

Comment:
Here, you shadow the already imported `gpg` module...

Reply:
unintentional, see previous comment - shall un-shadow.
Thanks for catching this.